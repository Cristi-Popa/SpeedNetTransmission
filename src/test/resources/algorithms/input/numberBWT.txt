aaaaa0bbbbbb74445559991222