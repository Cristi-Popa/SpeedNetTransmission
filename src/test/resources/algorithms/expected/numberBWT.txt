20|1a1|91|12|21|73|42|51b2|91|54a1|25b1|0